modules/programs/xplr.nix

```
@@ -35,15 +52,20 @@ in {
     package = mkPackageOption pkgs "xplr" { };
 
     plugins = mkOption {
-      type = with types; nullOr (listOf (either package str));
-      default = [ ];
-      defaultText = literalExpression "[]";
+      type = with types; nullOr (attrsOf (either package str));
+      default = { };
+      defaultText = literalExpression "{ }";
       description = ''
-        Plugins to be added to your configuration file.
+        An attribute set of plugin paths to be added to the [package.path]<https://www.lua.org/manual/5.4/manual.html#pdf-package.path> of the {file}`~/config/xplr/init.lua` configuration file.
```

Perhaps you mean
```suggestion
        An attribute set of plugin paths to be added to the [package.path](https://www.lua.org/manual/5.4/manual.html#pdf-package.path) of the {file}`~/config/xplr/init.lua` configuration file.
```
?

Also, I would suggest wrapping the line at roughly 80 char width.

---

Thank you. Resolved by latest force push.